applications/mne_analyze/plugins/annotationmanager/annotationmanager.cpp

```diff
@@ -197,6 +203,7 @@ void AnnotationManager::onModelChanged(QSharedPointer<ANSHAREDLIB::AbstractModel
         emit newFiffParamsAvailable(pFiffRawModel->absoluteFirstSample(),
                                     pFiffRawModel->absoluteLastSample(),
                                     pFiffRawModel->getFiffInfo()->sfreq);
+        emit newFiffRawViewModel(pFiffRawModel);
```

Comment:
MAybe it would be a good idea to merge `newFiffParamsAvailable` and `newFiffRawViewModel`. The receiver of `newFiffParamsAvailable` could just use `pFiffRawModel` to access `absoluteFirstSample`, `absoluteLastSample` and `getFiffInfo()->sfreq`.

Reply:
Combined functions. Got rid of newFiffParamsAvailable signal, changed passFiffParams slot to a private function that gets called by the onNewFiffRawViewModel slot that receives the newFiffRawViewModel signal.